### PR TITLE
added content feature scenario using gherkin

### PR DESCRIPTION
**Pull request by pritalpatel**
Work in progress.

### CODE REVIEW

File: tests/acceptance/content.feature [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Feature: Content
+  In order to manage content articles in the web
+  As an owner
+  I need to be able to create and modify content articles
+
+  Background:
+    Given I am logged in at Joomla administrator as "admin" with password "admin"
+    And I open administrator dashboard page
+
+  Scenario: create featured content
```
Comment (nibra) [inline tests/acceptance/content.feature]: Better: Feature an Article

File: tests/acceptance/content.feature [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    Given there is a content article with title "My first Joomla Content"
+    When I feature the content with title "My first Joomla Content"
+    Then I should see the published article "My first Joomla Content" in the joomla home page
+
+  Scenario: Add New Article
+    Given there is a content article with title "My first Joomla Article"
+    When I feature the content with title "My first Joomla Article"
+    Then I should see the published article "My first Joomla Article" in the joomla home page
```
Comment (nibra) [inline tests/acceptance/content.feature:14]: This is the same as above?!

File: tests/acceptance/content.feature [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  Scenario: Edit Article
+    Given there is a list of  content article with title
+    And I select the content article for edit with title "My first Joomla Article"
+    When I feature the content with title "My first Joomla Article"
+    Then I should see the published article "My first Joomla Article" in the joomla home page
+
+  Scenario: Unpublish  Article
+    Given there is a list of  content article with title
+    And I select the content article for unpublish  with title "My first Joomla Article"
```
Comment (nibra) [inline tests/acceptance/content.feature]: This line is not needed, it is part of the nect line already.

File: tests/acceptance/content.feature [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  In order to manage content articles in the web
+  As an owner
+  I need to be able to create and modify content articles
+
+  Background:
+    Given I am logged in at Joomla administrator as "admin" with password "admin"
+    And I open administrator dashboard page
+
+  Scenario: create featured content
+    Given there is a content article with title "My first Joomla Content"
```
Comment (javigomez) [inline tests/acceptance/content.feature]: I agree with Niels previous comment. according to your escenario, `Given that the "article" already exist` you will be "featuring the article" and not "creating a featured article". In the scenario we are not testing the article creation, we asume the article exist, we just test the `feature` feature.

File: tests/acceptance/content.feature [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    Scenario: Modify an article
+      Given I select the content article with title "My first Joomla Article"
+      And change the title "My Joomla Article"
+      When I try to post "My Joomla Article"
+      Then I should see the "Article successfully saved"
+      And published article "My Joomla Article" in the joomla home page
+
+  Scenario: Unpublish an article
+      Given I have article with name "My Joomla Article"
+      When I un publish "My Joomla Article"
```
Comment (pritalpatel) [inline tests/acceptance/content.feature:25]: Fix typo here, `un publish` should be `unpublish`

File: tests/acceptance/content.feature [hunk 6]
```diff
@@ -0,0 +1,37 @@
+Feature: content
+  In order to manage content article in the web
+  As an owner
+  I need to create modify delete and un public content article
```
Comment (pritalpatel) [inline tests/acceptance/content.feature:4]: Typo here: `un public` => `unpublish`

File: tests/acceptance/content.feature [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+      When I try to post "My Joomla Article"
+      Then I should see the "Article successfully saved"
+      And published article "My Joomla Article" in the joomla home page
+
+  Scenario: Unpublish an article
+      Given I have article with name "My Joomla Article"
+      When I un publish "My Joomla Article"
+      Then I can not see the article"My Joomla Article" in the joomla home page
+
+  Scenario: Remove an article
```
Comment (javigomez) [inline tests/acceptance/content.feature:28]: we will need to "trash":

screen shot 2016-05-12 at 16 08 28

And then we need also to empty the trash when the article is already in the trash:

capture 2016-05-04 at 16 10 01

File: tests/acceptance/content.feature [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+      Given I select the content article with title "My first Joomla Article"
+      And change the title "My Joomla Article"
+      When I try to post "My Joomla Article"
+      Then I should see the "Article successfully saved"
+      And published article "My Joomla Article" in the joomla home page
+
+  Scenario: Unpublish an article
+      Given I have article with name "My Joomla Article"
+      When I un publish "My Joomla Article"
+      Then I can not see the article"My Joomla Article" in the joomla home page
```
Comment (nibra) [inline tests/acceptance/content.feature:26]: Missing space after 'article'

**Pull request merged**